include/deal.II/matrix_free/portable_matrix_free.templates.h

```diff
@@ -772,7 +772,7 @@ namespace Portable
           MemorySpace::Default::kokkos_space::execution_space exec;
           Kokkos::TeamPolicy<
             MemorySpace::Default::kokkos_space::execution_space>
-            team_policy(exec, n_cells[color], Kokkos::AUTO);
+            team_policy(exec, n_cells[color], this->team_size);
```

Comment:
This makes assumptions of Kokkos internals mappping `Kokkos::AUTO` to `-1` for some backends.
This won't work for `OpenMP` for example. I'd rather create the policy beforehand using
```C++
using TeamPolicy = Kokkos::TeamPolicy<MemorySpace::Default::kokkos_space::execution_space>;
auto team_policy = (team_size == -1) ? 
  TeamPolicy(exec, n_cells[color], Kokkos::AUTO) : 
  TeamPolicy(exec, n_cells[color], team_size);
```